README.txt

```diff
@@ -217,7 +217,7 @@ Options
 
    Files, directories, function names, and class names that match this
    regular expression are considered tests.  Default:
-   (?:^|[b_./-])[Tt]est [NOSE_TESTMATCH]
+   (?:\b|_)[Tt]est [NOSE_TESTMATCH]
```

Comment:
You edited this by hand, but it's a generated file.  Run `make -C doc readme` to update it correctly.